extensions/wikia/Email/EmailController.class.php

```diff
@@ -72,12 +75,12 @@ public function init() {
 				$this->getVal( 'replyToName', $noReplyName )
 			);
 
-			$fromAddress = $this->getVal( 'fromAddress', '' );
+			$fromAddress = $this->getVal( 'fromAddress', $this->wg->PasswordSender );
```

Comment:
fromAddress and fromName were originally being passed into the controller b/c the first set of emails we did (the watched article emails) had confusing and custom logic for these values. Those emails will still pass in custom values for these fields, but most other emails default to using `$wgPasswordSender` and `$wgPasswordSenderName` so we can just make those the defaults.